### PR TITLE
Bridge Refactoring

### PR DESCRIPTION
**Pull request by gre**
Big refactoring on the bridge that allow more code sharing and a better bridge interface.

### Type

Refactoring

### Context

LL-1576

### Parts of the app affected / Test plan

Everything.
Only test the existing coins to not create any regression. especially on the Send flow.

### CODE REVIEW

File: static/i18n/en/app.json [hunk 1]
```diff
@@ -1220,6 +1220,9 @@
     "FeeRequired": {
       "title": "Fees are required"
     },
+    "GasLessThanEstimate": {
+      "title": "Provided gas limit is less than the estimated"
```
Comment (gre) [inline static/i18n/en/app.json]: @dasilvarosa can you think of a better wording?
it's a warning that displays under the gas limit field when user lower the gas limit under the safe estimated one. (setting that low value is likely to fail the transaction)

if we can not say "gas limit" in the wording it's better so we can later reuwe this concept. if you think we need to mention it, still fine.

Comment (dasilvarosa) [reply static/i18n/en/app.json]: This may be too low, please increase

Comment (gre) [reply static/i18n/en/app.json]: thanks

File: src/components/AccountPage/TokensList.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
```
Comment (juan-cortes) [inline src/components/AccountPage/TokensList.js]: Maybe rename this kind of files to `SubAccountList` then

Comment (gre) [reply src/components/AccountPage/TokensList.js]: true. we'll do that when we rework the UI/UX with design team. to also generalize that UI on their side.

**Pull request merged**